modules/test/test-howtos.bats

```diff
@@ -58,17 +58,22 @@ load 'test/test_helper.bash'
 
 @test "[howtos] - search.js" {
     runExample $HOWTOS_DIR search.js
-    assert_success
+    # This whole example is going to fail due to the below bug:
+    # https://issues.couchbase.com/browse/JSCBC-1044
+    assert_failure
+
+    # TODO: Uncomment (lines 66-76) when JSCBC-1044 is fixed
```

Comment:
might be worth adding that as diagnostic output, so we continue to see it?

I use `diag` https://github.com/couchbaselabs/try-cb-test/blob/main/test-helpers.bash#L66-L70
which makes sure it's commented correctly.

Reply:
Yeah useful output, let me add.